### ISSUE TITLE
Accessibility - impossible to open or close Bellows without a mouse/headers are not focusable

### ISSUE THREAD

**Issue by philwolstenholme** [mobify/bellows#67]
If you are navigating a page that uses Bellows with your keyboard (by using the tab key to move between focusable elements, and the space or enter key to interact with these elements) then it is impossible to open/close any contracted Bellows items as the headers are not given a tabindex to make them focusable.

http://heydonworks.com/practical_aria_examples/#progressive-collapsibles is a good example of a keyboard navigable accordion.

**Comment by philwolstenholme** [comment]
A partial fix for this is to make your Bellows header a `<button>` element. Unlike `<a>` elements Bellows will work with buttons as headers, and buttons are focusable, and pass their keyboard events on as clicks by the looks of things.
